### PR TITLE
Fix release build

### PR DESCRIPTION
**Pull request by conniey**
Updated the current build because it is using deprecated build tasks.  This fixes all the FxCop breaks because of the update.

/cc: @twsouthwick

### CODE REVIEW

File: src/ApiPort.VisualStudio.Common/AssemblyFile.cs [hunk 1]
```diff
@@ -51,8 +51,7 @@ public int Compare(IAssemblyFile x, IAssemblyFile y)
                 return y == null ? 0 : -1;
             }
 
-
-            return x.Name.CompareTo(y?.Name);
+            return string.CompareOrdinal(x.Name, y?.Name);
```
Comment (twsouthwick) [inline src/ApiPort.VisualStudio.Common/AssemblyFile.cs]: cool! I've always done `StringComparer.Ordinal.CompareTo(x,y)` - this API is half the length :)

File: src/ApiPort.VisualStudio.Common/DefaultProjectBuilder.cs [hunk 2]
```diff
@@ -22,20 +22,21 @@ namespace ApiPortVS
...
             IVSThreadingService threadingService,
             IProjectMapper projectMapper)
         {
-            _projectMapper = projectMapper;
+            ProjectMapper = projectMapper;
             _threadingService = threadingService;
             _buildManager = buildManager;
         }
 
+        public IProjectMapper ProjectMapper { get; }
```
Comment (twsouthwick) [inline src/ApiPort.VisualStudio.Common/DefaultProjectBuilder.cs]: does this need to be public or is protected good enough?

File: src/ApiPort.VisualStudio/Utils/TargetPlatformDisplayNameConverter.cs [hunk 3]
```diff
@@ -20,7 +20,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 
             if (target.AlternativeNames.Count > 0)
             {
-                return $"{target.Name} ({string.Join(", ", target.AlternativeNames)})";
+                return FormattableString.Invariant($"{target.Name} ({string.Join(", ", target.AlternativeNames)})");
```
Comment (twsouthwick) [inline src/ApiPort.VisualStudio/Utils/TargetPlatformDisplayNameConverter.cs]: you can always do `using static System.FormattableString` then just call `Invariant($"...")` - not a big deal, though

File: src/Microsoft.Fx.Portability.Offline/Data.cs [hunk 4]
```diff
@@ -92,7 +93,7 @@ private static Stream OpenFileOrResource(string path)
 
                 if (stream == null)
                 {
-                    throw new PortabilityAnalyzerException($"Could not find data file '{path}'. Either ensure the file is accessible during build or place it next to ApiPort at runtime.");
+                    throw new PortabilityAnalyzerException(((FormattableString)$"Could not find data file '{path}'. Either ensure the file is accessible during build or place it next to ApiPort at runtime.").ToString(CultureInfo.CurrentCulture));
```
Comment (twsouthwick) [inline src/Microsoft.Fx.Portability.Offline/Data.cs]: Why not use the `FormattableString.Invariant` here?

File: src/ApiPort.VisualStudio.Common/DefaultProjectBuilder.cs (hunk 2 above)
Comment (conniey) [inline src/ApiPort.VisualStudio.Common/DefaultProjectBuilder.cs]: Fixed to protected

File: src/ApiPort.VisualStudio/Utils/TargetPlatformDisplayNameConverter.cs (hunk 3 above)
Comment (conniey) [inline src/ApiPort.VisualStudio/Utils/TargetPlatformDisplayNameConverter.cs]: Can you do that for an abstract class? `public abstract class FormattableString`.

File: src/Microsoft.Fx.Portability.Offline/Data.cs (hunk 4 above)
Comment (conniey) [inline src/Microsoft.Fx.Portability.Offline/Data.cs]: Fixed

Comment (conniey) [inline src/Microsoft.Fx.Portability.Offline/Data.cs]: Changed.

**Pull request merged**